app/views/jobseekers/job_applications/build/personal_statement.html.slim

```diff
@@ -8,6 +8,19 @@
     h1.govuk-heading-l = t(".heading")
     p.govuk-body = t(".description")
 
+    - if job_application.vacancy.documents.any?
+      = govuk_notification_banner(title: t(".banner_title")) do |banner|
+        / added the following line because we need at least one heading for the component to be rendered
+        - banner.add_heading
+        = t(".banner_description_html", job_link: govuk_link_to(job_application.vacancy.job_title, job_path(job_application.vacancy), class: "document-link-component__link", target: "_blank"))
```

Comment:
Do we need this class on the job title link?

Reply:
nope, removed